### DISCUSSION TITLE
Redash not actively maintained. Active fork? Migration to alternatives experience?

### DISCUSSION THREAD

**Discussion by fragkakis** [getredash/redash#5954] [Q&A]
Hello,

it is evident that the opensource version of Redash is no longer being actively developed, below is the graph from the [contributors page](https://github.com/getredash/redash/graphs/contributors).

I have the following questions:
* Is there an actively developed fork of Redash?
* Has anyone migrated to alternatives, such as [Metabase](https://github.com/metabase/metabase) or [Superset](https://github.com/apache/superset) to share their experience?

Thank you!

<img width="910" alt="image" src="https://user-images.githubusercontent.com/222875/228771948-1c328e02-77c6-4a1f-bbf6-dd5b7f886b24.png">

**Comment by arikfr** [comment]
This was previously discussed at: https://github.com/getredash/redash/issues/5891, but now I have a response: https://github.com/getredash/redash/discussions/5962.

Of course people are welcome to migrate to other projects, but I would encourage anyone considering actively developing a fork to consider contributing to this repository instead.

**Reply by fragkakis** [reply]
Thank you for clearing things out @arikfr. 

Lots of users (myself included) love redash with a passion, and we want to see it evolve. The fact that we are still with it although it has not progressed speaks volumes of how well rounded and well thought out it is.

I hope things work out for Redash for the best.
